### ISSUE TITLE
pandas 0.24.1 testsuite failure with new numpy (

### ISSUE THREAD

**Issue by LocutusOfBorg** [pandas-dev/pandas#25322]
```
../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_finder_daily FAILED
../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_finder_quarterly FAILED
../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_finder_monthly FAILED
../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_finder_monthly_long PASSED
../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_finder_annual FAILED
../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_finder_minutely PASSED
../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_finder_hourly PASSED
../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_gaps FAILED
../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_gap_upsample FAILED
[...]

=================================== FAILURES ===================================
_________________________ TestTSPlot.test_finder_daily _________________________

self = <pandas.tests.plotting.test_datetimelike.TestTSPlot object at 0xc9b0e30c>

    @pytest.mark.slow
    def test_finder_daily(self):
        day_lst = [10, 40, 252, 400, 950, 2750, 10000]
    
        if (self.mpl_ge_3_0_0 or not self.mpl_ge_2_0_1
                or (self.mpl_ge_2_1_0 and not self.mpl_ge_2_2_2)):
            # 2.0.0, 2.2.0 (exactly) or >= 3.0.0
            xpl1 = xpl2 = [Period('1999-1-1', freq='B').ordinal] * len(day_lst)
        else:  # 2.0.1, 2.1.0, 2.2.2, 2.2.3
            xpl1 = [7565, 7564, 7553, 7546, 7518, 7428, 7066]
            xpl2 = [7566, 7564, 7554, 7546, 7519, 7429, 7066]
    
        rs1 = []
        rs2 = []
        for i, n in enumerate(day_lst):
            rng = bdate_range('1999-1-1', periods=n)
            ser = Series(np.random.randn(len(rng)), rng)
            _, ax = self.plt.subplots()
            ser.plot(ax=ax)
            xaxis = ax.get_xaxis()
            rs1.append(xaxis.get_majorticklocs()[0])
    
            vmin, vmax = ax.get_xlim()
            ax.set_xlim(vmin + 0.9, vmax)
            rs2.append(xaxis.get_majorticklocs()[0])
            self.plt.close(ax.get_figure())
    
>       assert rs1 == xpl1
E       assert [7566, 7566, ...66, 7566, ...] == [7565, 7564, 7...18, 7428, ...]
E         At index 0 diff: 7566 != 7565
E         Full diff:
E         - [7566, 7566, 7566, 7566, 7566, 7566, 7566]
E         + [7565, 7564, 7553, 7546, 7518, 7428, 7066]

../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py:425: AssertionError
_______________________ TestTSPlot.test_finder_quarterly _______________________

self = <pandas.tests.plotting.test_datetimelike.TestTSPlot object at 0xa48b6fcc>

    @pytest.mark.slow
    def test_finder_quarterly(self):
        yrs = [3.5, 11]
    
        if (self.mpl_ge_3_0_0 or not self.mpl_ge_2_0_1
                or (self.mpl_ge_2_1_0 and not self.mpl_ge_2_2_2)):
            # 2.0.0, 2.2.0 (exactly) or >= 3.0.0
            xpl1 = xpl2 = [Period('1988Q1').ordinal] * len(yrs)
        else:  # 2.0.1, 2.1.0, 2.2.2, 2.2.3
            xpl1 = [68, 68]
            xpl2 = [72, 68]
    
        rs1 = []
        rs2 = []
        for i, n in enumerate(yrs):
            rng = period_range('1987Q2', periods=int(n * 4), freq='Q')
            ser = Series(np.random.randn(len(rng)), rng)
            _, ax = self.plt.subplots()
            ser.plot(ax=ax)
            xaxis = ax.get_xaxis()
            rs1.append(xaxis.get_majorticklocs()[0])
    
            (vmin, vmax) = ax.get_xlim()
            ax.set_xlim(vmin + 0.9, vmax)
            rs2.append(xaxis.get_majorticklocs()[0])
            self.plt.close(ax.get_figure())
    
>       assert rs1 == xpl1
E       assert [72, 72] == [68, 68]
E         At index 0 diff: 72 != 68
E         Full diff:
E         - [72, 72]
E         + [68, 68]

../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py:455: AssertionError
________________________ TestTSPlot.test_finder_monthly ________________________

self = <pandas.tests.plotting.test_datetimelike.TestTSPlot object at 0xc45a44cc>

    @pytest.mark.slow
    def test_finder_monthly(self):
        yrs = [1.15, 2.5, 4, 11]
    
        if (self.mpl_ge_3_0_0 or not self.mpl_ge_2_0_1
                or (self.mpl_ge_2_1_0 and not self.mpl_ge_2_2_2)):
            # 2.0.0, 2.2.0 (exactly) or >= 3.0.0
            xpl1 = xpl2 = [Period('Jan 1988').ordinal] * len(yrs)
        else:  # 2.0.1, 2.1.0, 2.2.2, 2.2.3
            xpl1 = [216, 216, 204, 204]
            xpl2 = [216, 216, 216, 204]
    
        rs1 = []
        rs2 = []
        for i, n in enumerate(yrs):
            rng = period_range('1987Q2', periods=int(n * 12), freq='M')
            ser = Series(np.random.randn(len(rng)), rng)
            _, ax = self.plt.subplots()
            ser.plot(ax=ax)
            xaxis = ax.get_xaxis()
            rs1.append(xaxis.get_majorticklocs()[0])
    
            vmin, vmax = ax.get_xlim()
            ax.set_xlim(vmin + 0.9, vmax)
            rs2.append(xaxis.get_majorticklocs()[0])
            self.plt.close(ax.get_figure())
    
>       assert rs1 == xpl1
E       assert [216, 216, 216, 216] == [216, 216, 204, 204]
E         At index 2 diff: 216 != 204
E         Full diff:
E         - [216, 216, 216, 216]
E         ?             ^^   ^^
E         + [216, 216, 204, 204]
E         ?             ^^   ^^

../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py:485: AssertionError
________________________ TestTSPlot.test_finder_annual _________________________

self = <pandas.tests.plotting.test_datetimelike.TestTSPlot object at 0xc7d841cc>

    @pytest.mark.slow
    def test_finder_annual(self):
        if (self.mpl_ge_3_0_0 or not self.mpl_ge_2_0_1
                or (self.mpl_ge_2_1_0 and not self.mpl_ge_2_2_2)):
            # 2.0.0, 2.2.0 (exactly) or >= 3.0.0
            xp = [1987, 1988, 1990, 1990, 1995, 2020, 2070, 2170]
        else:  # 2.0.1, 2.1.0, 2.2.2, 2.2.3
            xp = [1986, 1986, 1990, 1990, 1995, 2020, 1970, 1970]
    
        xp = [Period(x, freq='A').ordinal for x in xp]
        rs = []
        for i, nyears in enumerate([5, 10, 19, 49, 99, 199, 599, 1001]):
            rng = period_range('1987', periods=nyears, freq='A')
            ser = Series(np.random.randn(len(rng)), rng)
            _, ax = self.plt.subplots()
            ser.plot(ax=ax)
            xaxis = ax.get_xaxis()
            rs.append(xaxis.get_majorticklocs()[0])
            self.plt.close(ax.get_figure())
    
>       assert rs == xp
E       assert [17, 18, 20, 20, 25, 50, ...] == [16L, 16L, 20L... 25L, 50L, ...]
E         At index 0 diff: 17 != 16L
E         Full diff:
E         - [17, 18, 20, 20, 25, 50, 100, 200]
E         + [16L, 16L, 20L, 20L, 25L, 50L, 0L, 0L]

../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py:518: AssertionError
_____________________________ TestTSPlot.test_gaps _____________________________

self = <pandas.tests.plotting.test_datetimelike.TestTSPlot object at 0xbf3dfb8c>

    @pytest.mark.slow
    def test_gaps(self):
        ts = tm.makeTimeSeries()
        ts[5:25] = np.nan
        _, ax = self.plt.subplots()
        ts.plot(ax=ax)
        lines = ax.get_lines()
        assert len(lines) == 1
        line = lines[0]
        data = line.get_xydata()
    
        if (self.mpl_ge_3_0_0 or not self.mpl_ge_2_0_1
                or (self.mpl_ge_2_1_0 and not self.mpl_ge_2_2_2)):
            # 2.0.0, 2.2.0 (exactly) or >= 3.0.0
            data = np.ma.MaskedArray(data, mask=isna(data), fill_value=np.nan)
    
>       assert isinstance(data, np.ma.core.MaskedArray)
E       AssertionError: assert False
E        +  where False = isinstance(array([[ 7.82700000e+03, -1.02081446e+00],\n       [ 7.82800000e+03,  8.0754728...[ 7.85500000e+03,  3.90151635e-01],\n       [ 7.85600000e+03, -8.36108671e-01]]), <class 'numpy.ma.core.MaskedArray'>)
E        +    where <class 'numpy.ma.core.MaskedArray'> = <module 'numpy.ma.core' from '/usr/lib/python2.7/dist-packages/numpy/ma/core.pyc'>.MaskedArray
E        +      where <module 'numpy.ma.core' from '/usr/lib/python2.7/dist-packages/numpy/ma/core.pyc'> = <module 'numpy.ma' from '/usr/lib/python2.7/dist-packages/numpy/ma/__init__.pyc'>.core
E        +        where <module 'numpy.ma' from '/usr/lib/python2.7/dist-packages/numpy/ma/__init__.pyc'> = np.ma

../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py:564: AssertionError
_________________________ TestTSPlot.test_gap_upsample _________________________

self = <pandas.tests.plotting.test_datetimelike.TestTSPlot object at 0xc9f115ac>

    @pytest.mark.slow
    def test_gap_upsample(self):
        low = tm.makeTimeSeries()
        low[5:25] = np.nan
        _, ax = self.plt.subplots()
        low.plot(ax=ax)
    
        idxh = date_range(low.index[0], low.index[-1], freq='12h')
        s = Series(np.random.randn(len(idxh)), idxh)
        s.plot(secondary_y=True)
        lines = ax.get_lines()
        assert len(lines) == 1
        assert len(ax.right_ax.get_lines()) == 1
    
        line = lines[0]
        data = line.get_xydata()
        if (self.mpl_ge_3_0_0 or not self.mpl_ge_2_0_1
                or (self.mpl_ge_2_1_0 and not self.mpl_ge_2_2_2)):
            # 2.0.0, 2.2.0 (exactly) or >= 3.0.0
            data = np.ma.MaskedArray(data, mask=isna(data), fill_value=np.nan)
    
>       assert isinstance(data, np.ma.core.MaskedArray)
E       AssertionError: assert False
E        +  where False = isinstance(array([[ 2.63016000e+05, -8.71992271e-01],\n       [ 2.63040000e+05,  2.4476855...[ 2.63928000e+05, -1.46319072e-01],\n       [ 2.63952000e+05,  1.57315404e+00]]), <class 'numpy.ma.core.MaskedArray'>)
E        +    where <class 'numpy.ma.core.MaskedArray'> = <module 'numpy.ma.core' from '/usr/lib/python2.7/dist-packages/numpy/ma/core.pyc'>.MaskedArray
E        +      where <module 'numpy.ma.core' from '/usr/lib/python2.7/dist-packages/numpy/ma/core.pyc'> = <module 'numpy.ma' from '/usr/lib/python2.7/dist-packages/numpy/ma/__init__.pyc'>.core
E        +        where <module 'numpy.ma' from '/usr/lib/python2.7/dist-packages/numpy/ma/__init__.pyc'> = np.ma

../debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_datetimelike.py:630: AssertionError
=============================== warnings summary ===============================
debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/extension/test_integer.py::TestMethods::()::test_combine_le[Int8Dtype]
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/extension/base/methods.py:153: RuntimeWarning: invalid value encountered in greater_equal
    expected = pd.Series([a <= val for a in list(orig_data1)])

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/frame/test_constructors.py::TestDataFrameConstructors::()::test_constructor_dict_nan_tuple_key[nan0]
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/core/common.py:207: RuntimeWarning: invalid value encountered in less
    return sorted(listed)

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/groupby/test_filters.py::test_filter_and_transform_with_non_unique_float_index
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/core/indexes/base.py:2656: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
    return self._engine.get_loc(key)

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_frame.py::TestDataFramePlots::()::test_logscales
  /usr/lib/python2.7/dist-packages/matplotlib/axes/_base.py:3157: UserWarning: Attempted to set non-positive xlimits for log-scale axis; invalid limits will be ignored.
    'Attempted to set non-positive xlimits for log-scale axis; '

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_frame.py::TestDataFramePlots::()::test_errorbar_plot
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/plotting/_core.py:1798: UserWarning: To output multiple subplots, the figure containing the passed axes is being cleared
    plot_obj.generate()
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/plotting/_core.py:1798: UserWarning: To output multiple subplots, the figure containing the passed axes is being cleared
    plot_obj.generate()
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/plotting/_core.py:1798: UserWarning: To output multiple subplots, the figure containing the passed axes is being cleared
    plot_obj.generate()

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_frame.py::TestDataFramePlots::()::test_errorbar_timeseries
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/plotting/_core.py:1798: UserWarning: To output multiple subplots, the figure containing the passed axes is being cleared
    plot_obj.generate()
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/plotting/_core.py:1798: UserWarning: To output multiple subplots, the figure containing the passed axes is being cleared
    plot_obj.generate()
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/plotting/_core.py:1798: UserWarning: To output multiple subplots, the figure containing the passed axes is being cleared
    plot_obj.generate()

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_hist_method.py::TestSeriesPlots::()::test_hist_legacy
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_hist_method.py::TestDataFramePlots::()::test_tight_layout
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_hist_method.py:253: UserWarning: To output multiple subplots, the figure containing the passed axes is being cleared
    _check_plot_works(df.hist)

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_misc.py::TestSeriesPlots::()::test_autocorrelation_plot
  /usr/lib/python2.7/dist-packages/matplotlib/pyplot.py:969: UserWarning: Requested projection is different from current axis projection, creating new axis with requested projection.
    return gcf().gca(**kwargs)
  /usr/lib/python2.7/dist-packages/matplotlib/pyplot.py:969: UserWarning: Requested projection is different from current axis projection, creating new axis with requested projection.
    return gcf().gca(**kwargs)

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/plotting/test_series.py::TestSeriesPlots::()::test_hist_legacy
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
  /usr/lib/python2.7/dist-packages/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/scalar/timedelta/test_arithmetic.py::TestTimedeltaMultiplicationDivision::()::test_td_div_nan[nan1]
  /<<PKGBUILDDIR>>/debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/scalar/timedelta/test_arithmetic.py:334: RuntimeWarning: invalid value encountered in double_scalars
    result = td // nan

debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/tseries/offsets/test_offsets_properties.py::test_apply_index_implementations
  /usr/lib/python2.7/dist-packages/dateutil/zoneinfo/__init__.py:26: UserWarning: I/O error(2): No such file or directory
    warnings.warn("I/O error({0}): {1}".format(e.errno, e.strerror))

-- Docs: https://docs.pytest.org/en/latest/warnings.html
 6 failed, 43011 passed, 3393 skipped, 777 deselected, 767 xfailed, 21 xpassed, 27 warnings in 5632.65 seconds 
```
Do you have any clue for the errors above?
this is the full log
https://launchpadlibrarian.net/411242961/buildlog_ubuntu-disco-i386.pandas_0.24.1-0ubuntu1_BUILDING.txt.gz

**Comment by LocutusOfBorg** [comment]
s390x is failing 17 test
https://launchpadlibrarian.net/411290359/buildlog_ubuntu-disco-s390x.pandas_0.24.1-0ubuntu1_BUILDING.txt.gz

**Comment by LocutusOfBorg** [comment]
ppc64el 4 tests
https://launchpadlibrarian.net/411284902/buildlog_ubuntu-disco-ppc64el.pandas_0.24.1-0ubuntu1_BUILDING.txt.gz

**Comment by LocutusOfBorg** [comment]
armhf 5 tests
https://launchpadlibrarian.net/411297231/buildlog_ubuntu-disco-armhf.pandas_0.24.1-0ubuntu1_BUILDING.txt.gz

**Comment by LocutusOfBorg** [comment]
arm64 5 tests
https://launchpadlibrarian.net/411295011/buildlog_ubuntu-disco-arm64.pandas_0.24.1-0ubuntu1_BUILDING.txt.gz

**Comment by jbrockmendel** [comment]
Which new numpy?  IIRC 1.16.0 had some problems that quickly got fixed in 1.16.1.

**Comment by LocutusOfBorg** [comment]
Hello, I confirm we are using numpy 1.16.1
```
Get:278 http://ftpmaster.internal/ubuntu disco-proposed/main armhf python3-numpy armhf 1:1.16.1-1ubuntu1 [1889 kB]
```

**Comment by gfyoung** [comment]
cc @jreback

**Comment by mroeschke** [comment]
Sorry that this never got addressed, but pandas will be on 1.3.0 soon which is Python 3 only and a minimum version of numpy that is 1.17. Going to close since we don't support this version anymore but if you could run these again with the latest version of pandas and open an issue it would be appreciated.
